azext_iot/central/commands_api_token.py

```diff
@@ -27,7 +27,7 @@ def add_api_token(
 
     try:
         role = Role[role].value
-    except:
+    except Exception:
```

Comment:
Is there a specific exception or no? Is this more for linting/style?

Reply:
This could be prob be a keyerror but this PR is making the minimum change for anywhere we have bare except without reason to at least Exception, so bare excepts do not block things like interrupts.